kolibri/plugins/user_profile/assets/src/views/ChangeFacility/index.vue

```diff
@@ -118,6 +118,7 @@
           value: {
             facility: this.session.facility_id,
             username: this.session.username,
+            fullName: this.session.full_name,
```

Comment:
Not sure if we're needing this in the next steps, but it does not harm to keep it anyway

Reply:
It's used here https://github.com/learningequality/kolibri/pull/9576/files#diff-3481d531d29d62496d420d7a168388365376baf2c240b7547703c3e4fed49b51R92. Or did you mean some another future step different from this one?

Reply:
yes, I meant a future step, if it's not needed later, it does not need to be added to the machine context.

Reply:
You're right, it'll make more sense to rather reach to the session directly from the page. Will update.

Just to learn a bit about patterns for the future, I think that how often is data used is not the main criteria, and your comment is rather about that just in the context of the machine itself, it would be an extra field that's not being used in the machine anywhere so that's why it doesn't make much sense, is that right? Therefore it'd be fine to put something in the context even if it's used only once, but that "once" should happen in the machine (e.g. in `cond`)

Reply:
yes, I would not add it to the session either. In case other screens in the flow need it, I would add it to the context. My question was if other screens are going to need the fullName. If not, it can be just a data variable in the component.
Anything that needs to be used later, in a `cond` as you say, or in an `action`, should be in the context. Actually I see the context as a kind of vuex object, for anything the states can need.

Reply:
By session, I meant `session` getter, the same that's used in ChangeFacility/index.js

Reply:
I am not sure about the other screens, will have a look. Even if they needed it though, I think that they could too simply use `session` getter directly, rather than saving `session.full_name` to the machine's context only so those pages could get it, when there's no need to go this "longer" path to get to the data, especially when it's useless for the machine itself.